### PR TITLE
FightClub Improvement.

### PR DESCRIPTION
**Pull request by Specifer**
New commit for #346

FightClub saving pos data.

### CODE REVIEW

**Comment by RussellLVP** [comment]
This is probably fine, if we were to test it, but please make sure that these are two separate PRs. The two changes have nothing to do with each other :).

**Comment by Specifer** [comment]
Yes ok..

File: pawn/Resources/Games/FightClub.pwn [hunk 1]
```diff
@@ -1071,8 +1093,9 @@ CFightClub__EndMatch(matchid, deathPlayerId)
...
     if(Player(iPlayer1)->isConnected())
     {
-        SetPlayerVirtualWorld(iPlayer1, 0);
-        SetPlayerInterior(iPlayer1, 0);
+        SetPlayerPos(iPlayer1, FighterPos[iPlayer1][0], FighterPos[iPlayer1][1], FighterPos[iPlayer1][2]);
+        SetPlayerVirtualWorld(iPlayer1, fighterworld[iPlayer1]);
+        SetPlayerInterior(iPlayer1, fighterint[iPlayer1]);
         ResetWorldBounds(iPlayer1);
 
         if (iPlayer1 != deathPlayerId) {
```
Comment (RussellLVP) [inline pawn/Resources/Games/FightClub.pwn:1100]: There's a `SpawnPlayer` call immediately after here, wouldn't that break it?

The spectate change of this PR looks good, but I think that the adjustment for fighters is broken. I think that the existing `SetPlayerVirtualWorld` and `SetPlayerInterior` calls were also broken.

**Comment by Specifer** [comment]
Does this change work @RussellLVP?

**Comment by RussellLVP** [comment]
I am not sure. If I'm honest, I would rather that we drop this fix for now, as fight club will move to JavaScript in its entirety built on top of the `/match` system.

**Pull request closed without merging**